README.md

```diff
@@ -179,3 +160,18 @@ For testing:
   require a password by editing `/etc/postgresql/10/main/pg_hba.conf` and
   changing `md5` to `trust` for both the IPv4 and IPv6 local connections
   settings, and then restart postgres via `sudo systemctl restart postgresql`.
+
+### Testing
```

Comment:
@jonahkagan Perhaps add in somewhere in here that [the dependencies for cypress might need to be installed](https://docs.cypress.io/guides/getting-started/installing-cypress.html#System-requirements).

Reply:
was there something in particular you needed to install?

Reply:
I didn't notice what it was, but it installed something when I ran it when I was troubleshooting. So I actually don't know if it was part of what caused me problems or not, haha. I just thought it would likely be handy eventually for someone.